src/python/pants/backend/go/target_types.py

```diff
@@ -15,15 +15,60 @@ class GoPackageSources(GoSources):
 class GoImportPath(StringField):
     # TODO: Infer the import path from the closest ancestor `go_module` target once that target is supported.
     alias = "import_path"
-    help = "Import path in Go code to import this package."
+    help = "Import path in Go code to import this package or module."
+
+
+class GoPackageDependencies(Dependencies):
+    pass
 
 
 class GoPackage(Target):
     alias = "go_package"
-    core_fields = (*COMMON_TARGET_FIELDS, Dependencies, GoPackageSources, GoImportPath)
+    core_fields = (*COMMON_TARGET_FIELDS, GoPackageDependencies, GoPackageSources, GoImportPath)
     help = "A single Go package."
 
 
+# `go_module` target
+
+
+class GoModuleSources(Sources):
+    default = ("go.mod", "go.sum")
```

Comment:
SHould the user be allowed to override this? From what I understand in our discussions, this should be 1 per directory.

You can set `alias = "_sources"` if you want to hide the field from the user, i.e. not show up in `./pants help`.

Reply:
Done.